recipes/salmon/meta.yaml

```diff
@@ -1,15 +1,17 @@
-{% set version="1.7.0" %}
+{% set version="1.8.0" %}
 
 package:
   name: salmon
   version: {{ version }}
 
 build:
   number: 0
+  skip: True #[linux]
```

Comment:
Do you skip the Linux build? That might be dangerous for the next update. I propose to simply bump the version number, or add a comment to the skip.

Reply:
Hi @bgruening ; I don't! I disabled it because I was trying to fix a bug with OSX build, and wanted to save time. This isn't quite ready fir merge yet. I definitely want to re-enable linux.  Thanks for checking before merging.

Reply:
Ah ok, I renamed the PR into WIP, hope that is ok.

Reply:
Thanks!  I think once I re-enable the build it will be good to go.  Should I remove WIP before I merge then?

Reply:
Would be nice.